packages/kuma-gui/src/app/x/components/x-action-group/XActionGroup.vue

```diff
@@ -13,7 +13,7 @@
         :kpop-attributes="{
           placement: 'bottom-end',
         }"
-        width="150"
+        :width="width"
```

Comment:
Note: The width was hardcoded to 150. This caused that long labels of action items overflow the borders of the action group dropdown.

Reply:
Do you know what happens if we use `auto` for the default, and then don't use the 150 width at all anywhere? 

I'm guessing the 150 might have just come from where we took this from originally. Actually, hold on let me have a little snoop around I might be able to see why we did this. brb

Ideally we wouldn't have to specify sizes and the menu would just shrink/grow to contain the content.

Reply:
hmmm yeah ideally we need a min-width of `150`

## with 150
<img width="284" alt="Screenshot 2024-11-18 at 17 31 20" src="https://github.com/user-attachments/assets/13124469-f4fc-4bc5-88b6-d1095f072f09">

## without 150
<img width="209" alt="Screenshot 2024-11-18 at 17 31 31" src="https://github.com/user-attachments/assets/d84015d2-d1dd-454b-8c21-df81bd110a52">

Reply:
> hmmm yeah ideally we need a min-width of `150`

I don't think that this is a solution that is generic enough. If we would, for whatever reason, decide to rephrase the content to just `View` or add another dropdown somewhere with a very short action item label, the `150px` would probably too big again 🤔

Reply:
Ah think we crossed over, I just tried out your point 3 also:

```css
.x-action-group :deep(.popover-container) {
  min-width: 150px;
}
```

I also had to keep `width="auto"` on the KDropdown otherwise it seems to get set to `width: 200px`
 even though "The width of the dropdown body (defaults to auto)." (https://kongponents.konghq.com/components/dropdown.html#width)

I think we should go with that. There is no chance we'll have a i18n'ed GUI if we have hardcoded sizes (I think this originally came from elsewhere). I think if we go with point 3, we stay consistent with elsewhere, but things grow once we start properly i18n'ing things. WDYT?